src/firstvoices/backend/utils/CharacterSortOrderHelpers.py

```diff
@@ -0,0 +1,127 @@
+import re
+import unicodedata
+
+
+# From MTD processors
```

Comment:
For the benefit of our future selves, can you please include a more detailed comment about where this came from? Preferably with a permalink to the relevant line in a git repo